src/middlewares/__tests__/ssm.js

```diff
@@ -35,9 +36,27 @@ describe('🔒 SSM Middleware', () => {
     handler.use(ssm(middlewareOptions))
 
     const event = {}
-    handler(event, context, (error, response) => {
-      cb(error, {event, context, response})
+    let promise = Promise.resolve()
+    callbacks.forEach(cb => {
+      let context = {}
+      promise = promise.then(() => {
+        return new Promise((resolve, reject) => {
+          handler(event, context, (error, response) => {
+            try {
+              cb(error, {event, context, response})
+              resolve()
+            } catch (err) {
+              reject(err)
+            }
+          })
+        })
+      }).then(() => {
+        if (delay) {
+          return Promise.delay(delay)
```

Comment:
Are we adding `bluebird` only for this feature (`Promise.delay`)?

Can't we just do something like the following and get rid of `bluebird`?

```javascript
// ...
return new Promise((resolve, reject) => setTimeout(resolve, delay))
// ...
```

I know this is only for dev mode, but I would prefer to avoid to introduce new libs if they don't bring a substantial value.

Reply:
sure thing